Wikipedia/View Controllers/WebView/WebViewController.m

```diff
@@ -1059,6 +1057,10 @@ - (void)scrollViewDidScroll:(UIScrollView*)scrollView {
     CGFloat distanceScrolled     = self.scrollViewDragBeganVerticalOffset - scrollView.contentOffset.y;
     CGFloat fabsDistanceScrolled = fabs(distanceScrolled);
 
+    if (![self tocDrawerIsOpen]) {
```

Comment:
At this point, you can remove all the "tocDrawer" stuff totally. The no need to respond to changes. You can leave the "TOC" button unhooked, and I can hook it back up in my commit.